### PR TITLE
Optimize: correction code specification

### PR DESCRIPTION
**Pull request by demoManito**
#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
The coding specification of go is followed

case:
empty slices: https://github.com/golang/go/wiki/CodeReviewComments#declaring-empty-slices
Interface checks: https://go.dev/doc/effective_go#blank_implements
if logic: https://go.dev/doc/effective_go#if
#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs
NONE
```

### CODE REVIEW

**Comment by demoManito** [comment]
@jsafrane Under the troublesome codereview, some code styles were corrected

File: pkg/volume/awsebs/attacher.go [hunk 1]
```diff
@@ -43,13 +43,13 @@ type awsElasticBlockStoreAttacher struct {
 	awsVolumes aws.Volumes
 }
 
-var _ volume.Attacher = &awsElasticBlockStoreAttacher{}
+var _ volume.Attacher = (*awsElasticBlockStoreAttacher)(nil)
```
Comment (jsafrane) [inline pkg/volume/awsebs/attacher.go:46]: We have this code all over Kubernetes. I'd prefer to have it consistent, so either change it everywhere in this repo or keep it as it is.

The other changes looks OK.

Comment (demoManito) [reply pkg/volume/awsebs/attacher.go:46]: Thanks for the reply, I will remove the interface check changes

**Comment by demoManito** [comment]
@jsafrane the interface check change has been deleted, please reivew it again

**Pull request merged**